### DISCUSSION TITLE
Newby still in diapers

### DISCUSSION THREAD

**Discussion by grasshopper48** [community/community#45277] [Other Feature Feedback, Questions, & Ideas]
Im here to locate me own Mac backup encrypted passwords and to learn how to help folks recover access to their forgotten social media accounts. not real familiar with coding but I've been teaching myself a little bit. idk even know where to begin

**Comment by jge162** [comment]
Any questions GitHub related?

**Comment by grasshopper48** [comment]
Yes I’d like to get into my I message through Mack but my girlfriend or ex
changed all my passwords and hers. I’d there a way to achieve this small
feat. Also I have backed up my iPhone to my Mac and the encrypted password
has been changed or I’m missing something.

On Tue, Jan 24, 2023 at 8:46 PM Jermyiah ***@***.***> wrote:

> Any questions GitHub related?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/45277#discussioncomment-4773017>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A5OGRAOURFPJBGEYG4UNQTLWUCHX5ANCNFSM6AAAAAAUFS352U>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
E

**Reply by jge162** [reply]
Can’t help u sorry

**Comment by grasshopper48** [comment]
No worries. Is there not any way to break my own passwords?

**Reply by jge162** [reply]
No idea sorry. Wrong forum. GitHub related help.

**Comment by grasshopper48** [comment]
I’m on phone with Apple. Hopefully they can help!!

**Comment by grasshopper48** [comment]
Understand . How to implement GitHub to use these suggestions to possibly
help me achieve this feat!
